shared/authplugin/src/main/java/io/pravega/auth/AuthHandler.java

```diff
@@ -69,7 +70,9 @@
     /**
      * Sets the configuration. If the auth handler needs to access the server configuration, it can be accessed though this var.
      *
-     * @param serverConfig The server configuration.
+     * @param properties The auth handler configuration.
      */
-    void initialize(ServerConfig serverConfig);
+    default void initialize(Properties properties) {
```

Comment:
lets keep the interface signature same. lets change ServerConfig to add a new API for getting properties.

Reply:
Sure, I've now modified the implementation, as suggested, to avoid the change in the interface here - handler.initialize() continues to take `ServerConfig` as input.